ts-client/src/dlmm/index.ts

```diff
@@ -3623,6 +3629,339 @@ export class DLMM {
     );
   }
 
+  /**
+   * The `seedLiquidity` function create multiple transactions, which each transaction consists of initialize position, and add liquidity instruction. The combination of liquidity deposited to the position will be the ilm / lfg curve
+   * @param
+   *
+   */
+  public async seedLiquidity(
+    owner: PublicKey,
+    operator: PublicKey,
+    feeOwner: PublicKey,
+    seedAmount: BN,
+    curvature: number,
+    minPrice: number,
+    maxPrice: number,
+    base: PublicKey
+  ): Promise<TransactionInstruction[][]> {
+    const toLamportMultiplier = new Decimal(
```

Comment:
Can we write a function to calculate amounts for each bin? so easier for unit test

Reply:
Updated